clusters.py

```diff
@@ -7,7 +9,21 @@
 of the user.
 """
 spreadsheet_v1_path = 'https://docs.google.com/spreadsheets/d/119_-qrfzGXwV1YBUJdBzvtAQTZnl-xwN7hD9FK5SWfU/edit#gid=84023254'
+# features_before = [ "AgeAtTestDate", "R250", "R500", "R1000", "R2000", "R3000", "R4000", "R6000", "R8000",
+#               "L250", "L500", "L1000", "L2000", "L3000", "L4000", "L6000", "L8000", "RBone500",
+#               "RBone1000", "RBone2000", "RBone4000", "LBone500", "LBone1000", "LBone2000", "LBone4000",
+#               "MonSNR_Score_R", "Word_Rec_Score_R", "MonSNR_Score_L", "Word_Rec_Score_L" ]
+
+duplicate_column_name_1: str = 'LBone2000'
```

Comment:
Change name_1 to name_v1 (it wasn't clear what the 1 meant the first time I read this.)